Diary/Diary/PriorityTask.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Diary
+{
+    internal class PriorityTask : RegularTask
+    {
+        internal string priority;
+
+        internal PriorityTask(string _name) : base(_name)
+        {
+            name = _name;
+        }
+
+        internal PriorityTask(string _name, string _priority) : base(_name)
+        {
+            name = _name;
+            priority = _priority;
+        }
+
+        internal PriorityTask(string _name, DateTime _dataTask, DateTime _timeTask) : base(_name, _dataTask, _timeTask)
+        {
+            name = _name;
+            dataTask = _dataTask;
+            timeTask = _timeTask;
+        }
+
+        internal PriorityTask(string _name, DateTime _dataTask, DateTime _timeTask, string _priority) : base(_name, _dataTask, _timeTask)
+        {
+            name = _name;
+            dataTask = _dataTask;
+            timeTask = _timeTask;
+            priority = _priority;
+        }
+
+        internal override string GetAlarm()
+        {
+            return ToString();
```

Comment:
`ToString` is supposed to just print task, like "<name>, <date> <time>", but this method should calculate the days diff. 
Now everything is mixed up.